src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -158,6 +158,14 @@ def __init__(
         google_api_key: Optional[str] = None,
         exchange_locked: Optional[bool] = None,
     ) -> None:
+        self.__cache_modifier = ""
+        if default_exchange and exchange_locked:
+            self.__cache_modifier += default_exchange.replace(" ", "_")
+        if fiat_priority:
+            if self.__cache_modifier:
+                self.__cache_modifier += "_"
+            self.__cache_modifier += "_".join(fiat_priority)
```

Comment:
Nit: simplification suggestion:
```suggestion
        exchange_cache_modifier = default_exchange.replace(" ", "_") if default_exchange and exchange_locked else ""
        fiat_priority_cache_modifier = fiat_priority if fiat_priority else ""
        self.__cache_modifier = "_".join([exchange_cache_modifier, fiat_priority_cache_modifier])
```